api/src/resources/controllers/Base.controller.ts

```diff
@@ -17,7 +17,9 @@ abstract class BaseController {
 
   constructor(model: Model<any>) {
     this._model = model;
-    console.log(`⚡ [Server]: {Controller} :: ${this.constructor.name} initialized...`);
+    logger.info(
```

Comment:
Can we change this to be
`this._logger = logger`

and also add the property above with the correct type?